cassandra/cqlengine/columns.py

```diff
@@ -168,6 +168,36 @@ def __init__(self,
         self.position = Column.instance_counter
         Column.instance_counter += 1
 
+    def __ne__(self, other):
+        if isinstance(other, Column):
+            return self.instance_counter != other.instance_counter
```

Comment:
These comparisons all use ``Column.instance_counter``, which is the same for all instances. Do you want to use ``self.position`` instead?